conf.py

```diff
@@ -66,3 +66,6 @@
         "image_dark": "media/releng-dark.png",
     },
 }
+
+# Set canonical URL from the Read the Docs Domain
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
```

Comment:
oops, need to import `os`. Also, we may need this bit?

# Tell Jinja2 templates the build is running on Read the Docs
if os.environ.get("READTHEDOCS", "") == "True":
    html_context["READTHEDOCS"] = True

Reply:
`import os`> oops indeed...

I'm not seeing any jinja2 templates in the repo, so I kind of skipped that bit on purpose.  Do you think we should add it anyway?  (Quite possible I'm misunderstanding how that gets used...)

Reply:
I think sphinx uses jinja2 for templating by default? I guess we're not making use of it though.